modules/envpackages.nix

```diff
@@ -29,7 +29,15 @@ in
       nettools # Needed for "hostname" in build/soong/ui/build/sandbox_linux.go
       procps # Needed for "ps" in build/envsetup.sh
     ]
-    (mkIf (config.androidVersion >= 10) [
+    (mkIf (config.androidVersion >= 12) [
+      freetype # Needed by jdk9 prebuilt
+      fontconfig
+
+      # Goldfish doesn't need py2 anymore in Android 12+!
```

Comment:
Okay not 100% sure how to trace this commit to Android version, but I think that's right